### ISSUE TITLE
Configuring IP-Address when starting as Admin

### ISSUE THREAD

**Issue by Alfrodo12** [Microsoft/WinAppDriver#58]
Hi Folks,

can anybody explain me how to configure another IP-address when starting the winappdriver.exe as Administrator?

Thanks a lot

Thomas

**Comment by Alfrodo12** [comment]
Thanks for adding the documentation in the readme.md. Everything is working now with my first remote agent in a VM on my local machine.

I´m testing now the diffeent API´s with a sample UWP app, because we want to define a test Scenario for a customer Windows 10 UWP app as next step.

Can you give me some advice for my upcoming questions:

1.  Are you sure that winappdriver will replace CodeUI Tests regarding Windows 10 UWP apps?
2. I saw in another ticket that Integration in TFS/VSO is on your backlog. Can you estimate a timeline?

Best regards

Thomas

**Comment by alakshmi1030** [comment]
1. Yes I am sure - the timeline is not set yet for when WinAppDriver will replace CodedUI
2. We are still working with the Visual Studio team to figure out the details of how WinAppDriver integration into VSO would work. As such we don't have a definitive timeline yet but we will update the community once this is ready to be released.
